src/Pact/Analyze/Model/Text.hs

```diff
@@ -126,13 +126,11 @@ showEvent ksProvs tags = \case
       [display mtAsserts tid (showAssert recov)]
     TraceAuth recov (_located -> tid) ->
       [display mtAuths tid (showAuth recov $ tid `Map.lookup` ksProvs)]
-    TraceBind (_located -> (vid, _, _)) ->
-      [display mtVars vid showVar]
     TraceSubpathStart _ ->
       [] -- not shown to end-users
-    TracePushScope _ ->
-      []
-    TracePopScope _ ->
+    TracePushScope _ _ (fmap (view (located.bVid)) -> vids) ->
```

Comment:
Can I request you give this view pattern a name or something? I'm having trouble reading it.

Reply:
I switched this to use a let-binding